multi-agent/tests/integration/endpoints/conftest.py

```
@@ -0,0 +1,105 @@
+        resources_service=resources_service,
+        builtin_resource_service=builtin_resource_service,
+        collaboration_service=collaboration_service,
+        admin_edit_lock_service=admin_edit_lock_service,
+        admin_config_reader=None,
+    )
+
+
+@pytest.fixture
+def app(container):
+    flask_app = Flask(__name__)
+    flask_app.secret_key = "test-secret"
```

Ruff (0.16.1)

[error] Possible hardcoded password assigned to: "secret_key"

(S105)